client/dashboard/me/notifications-extras/extras-toggle-card/test/index.test.tsx

```diff
@@ -0,0 +1,239 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { WPCOM_OPTION_KEYS, WPCOM_TITLES, WPCOM_DESCRIPTIONS } from '../../extras-config';
+import { ExtrasToggleCard } from '../index';
+import type { WpcomNotificationSettings } from '@automattic/api-core';
+
+const mockOnMutate = jest.fn();
+
+const defaultExtraSettings: Partial< WpcomNotificationSettings > = {
+	marketing: false,
+	research: false,
+	community: false,
+	promotion: false,
+	news: false,
+	digest: false,
+	reports: false,
+	news_developer: false,
+	scheduled_updates: false,
+};
+
+describe( 'ExtrasToggleCard', () => {
+	beforeEach( () => {
+		mockOnMutate.mockClear();
```

Comment:
Minor Note: Typically, it is more scalable to start with a default callback (onMutate) and redefine it only for the test that really requires it. The current approach is prone to false positives, just because. We forget to clean and reset the mock.